### PR TITLE
plan and articles

### PR DESCRIPTION
**Pull request by NathanLee3296**

### CODE REVIEW

File: src/Components/Articles.jsx
Comment (NathanLee3296) [inline src/Components/Articles.jsx:1]: Will rename to ArticleList

File: src/Components/Articles.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		getArticles().then((articles) => {
+			setArticleList(articles);
+			setIsLoading(false);
+		});
+	}, []);
+
+	return (
+		<section id="article-list">
+			<h2>Articles</h2>
+			{isLoading && <p>page loading</p>}
```
Comment (2202Hannah) [inline src/Components/Articles.jsx:20]: Nice work with your conditional rendering here and use of the `&&` shorthand 👍

File: src/Requests/makeRequests.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
 import axios from "axios"
```
Comment (2202Hannah) [inline src/Requests/makeRequests.js:3]: You could use axios.create to set up a base URL to use in each axios request in this file.

File: src/Components/Articles.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+			{!isLoading &&
+				articleList.map((article) => {
+					return <ArticleCard key={article.article_id} article={article} />;
```
Comment (2202Hannah) [inline src/Components/Articles.jsx:23]: Nice job extracting the code for ArticleCard in to a child component

**Review by 2202Hannah — APPROVED** [review]
Hi Nathan, great work on task 4. I noticed you are using utils functions to display the time data in a user friendly way which does make for a nice user experience, I am wondering if this might be better handled by a js library which you can install in your project (you can search for these on google).

**Review by 2202Hannah — COMMENTED** [review]
Great work on your plan too! I would like you to consider the following:
- Your articles list will retrieve data from the backend endpoint that includes a summary of the articles WITHOUT the article body
- You will need to make a different api call for a single article that includes the body, based on the article_id
- These two different types of article page should be separated in to different branches from app and would in fact by siblings to each other rather than a parent/child.

**Pull request merged**